homeassistant/components/binary_sensor/command_sensor.py

```diff
@@ -0,0 +1,82 @@
+"""
+homeassistant.components.binary_sensor.command_sensor
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Allows to configure custom shell commands to turn a value
+into a logical value for a binary sensor.
+"""
+import logging
+import subprocess
+from datetime import timedelta
+
+from homeassistant.const import CONF_VALUE_TEMPLATE
+from homeassistant.components.binary_sensor import BinarySensorDevice
+from homeassistant.component.sensor.command_sensor import CommandSensorData
```

Comment:
`components` should be plural.

Please make sure you test your code locally before pushing it out. This could have been caught easily.